util/execution/tpl.cpp

```diff
@@ -252,8 +252,8 @@ static void CompileAndRun(const std::string &source, const std::string &name = "
       parse_ms, typecheck_ms, codegen_ms, interp_exec_ms, adaptive_exec_ms, jit_exec_ms);
   txn_manager.Commit(txn, transaction::TransactionUtil::EmptyCallback, nullptr);
   catalog.TearDown();
-  gc.PerformGarbageCollection();
-  gc.PerformGarbageCollection();
+  da_manager.Process();
+  da_manager.Process();
```

Comment:
Doesn't your destructor handle this now?

Reply:
Yup, i'll modify that